<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Espace de travail :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="CheminEspaceTravail">
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>RÃ©sultat de la echerche</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget_Resultat"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Utilisateurlabel">
      <property name="text">
       <string>Utilisateur : </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuFiche_Technique">
    <property name="title">
     <string>Fiche Technique</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer_sous"/>
    <addaction name="separator"/>
    <addaction name="actionAjouter"/>
    <addaction name="actionSupprimer"/>
    <addaction name="separator"/>
    <addaction name="actionRechercher"/>
    <addaction name="actionTrier"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image processing</string>
    </property>
    <widget class="QMenu" name="menuLaplacian">
     <property name="title">
      <string>Laplacian</string>
     </property>
     <addaction name="actionSobel"/>
     <addaction name="actionPrwitt"/>
    </widget>
    <addaction name="actionHistogramme"/>
    <addaction name="actionConvolution"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionGradient"/>
    <addaction name="menuLaplacian"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPermitons"/>
    <addaction name="actionConsutation"/>
    <addaction name="separator"/>
    <addaction name="actionAdmnistrator"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuFiche_Technique"/>
   <addaction name="menuImage"/>
   <addaction name="menuSettings"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionOuvrir"/>
   <addaction name="actionEnregistrer_sous"/>
   <addaction name="actionAjouter"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRechercher"/>
   <addaction name="actionTrier"/>
   <addaction name="separator"/>
   <addaction name="actionSupprimer"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/RC/img/open.jpg</normaloff>:/RC/img/open.jpg</iconset>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/RC/img/save.jpg</normaloff>:/RC/img/save.jpg</iconset>
   </property>
   <property name="text">
    <string>Enregistrer sous</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/RC/img/close.jpg</normaloff>:/RC/img/close.jpg</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAjouter">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/RC/img/add.jpg</normaloff>:/RC/img/add.jpg</iconset>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="icon">
    <iconset>
     <normaloff>img/delete.jpg</normaloff>img/delete.jpg</iconset>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionPermitons">
   <property name="text">
    <string>Permitons</string>
   </property>
   <property name="toolTip">
    <string>Permitons</string>
   </property>
  </action>
  <action name="actionConsutation">
   <property name="text">
    <string>Consutation</string>
   </property>
  </action>
  <action name="actionAdmnistrator">
   <property name="text">
    <string>Admnistrator</string>
   </property>
  </action>
  <action name="actionHistogramme">
   <property name="text">
    <string>Histogramme</string>
   </property>
  </action>
  <action name="actionConvolution">
   <property name="text">
    <string>Convolution</string>
   </property>
  </action>
  <action name="actionGradient">
   <property name="text">
    <string>Gradient</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionPrwitt">
   <property name="text">
    <string>Prwitt</string>
   </property>
  </action>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionRechercher">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/RC/img/search.jpg</normaloff>:/RC/img/search.jpg</iconset>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
  </action>
  <action name="actionTrier">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/RC/img/trier.jpg</normaloff>:/RC/img/trier.jpg</iconset>
   </property>
   <property name="text">
    <string>Trier</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
